include/modem/nrf_modem_lib_trace.h

```diff
@@ -51,6 +51,18 @@ int nrf_modem_lib_trace_processing_done_wait(k_timeout_t timeout);
  */
 int nrf_modem_lib_trace_level_set(enum nrf_modem_lib_trace_level trace_level);
 
+#if CONFIG_NRF_MODEM_LIB_TRACE_BITRATE
```

Comment:
If we want to show this in the documentation: 
```suggestion
#if CONFIG_NRF_MODEM_LIB_TRACE_BITRATE  || defined(__DOXYGEN__)
```